CHANGES.rst

```diff
@@ -50,6 +50,8 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
+  - A deepcopy() method was added to models. [#6515]
```

Comment:
Nitpick: Double-backticks around `deepcopy()`.

Maybe further clarification that `deepcopy()` for single model is the same as `copy()` but not for compound? Or is such explanation more appropriate for the proper user-facing doc somewhere?